### DISCUSSION TITLE
should yarn generate also display _some_ of the info from yarn account

### DISCUSSION THREAD

**Discussion by austintgriffith** [scaffold-eth/scaffold-eth-2#504] [Ideas]
okay okay this one is real lazy, but it might be nice if, after a yarn generate, it displays the new address that was generated so it's faster to copy paste send without running yarn account

**Comment by carletex** [comment]
It was added here https://github.com/scaffold-eth/scaffold-eth-2/pull/485 (for hardhat). Let's add it for Foundry too! @technophile-04 @jrcarlos2000 

![Screenshot 2023-08-12 at 4 31 02 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/13369344-3680-4c86-8ee9-8d6195615592)

**Reply by technophile-04** [reply]
Its was added in`foundry` branch too 🙌 at https://github.com/scaffold-eth/scaffold-eth-2/commit/b3fb683bebea0d7f7d75066d7b6adb27196e75a4 also being available in CLI since (v18) for both hardhat and foundry 🙌
